en/docs/deploy/add-logs-for-tokens.md

```diff
@@ -9,16 +9,16 @@ disabled from appearing in the logs by setting each token to
 `         false        ` .
 
 ``` c#
-    UserClaims=true
-    UserIdToken=true
-    XACML_Request=true
-    XACML_Response=true
-    NTLM_Token=true
-    SAML_Assertion=true
-    SAML_Request=true
-    AccessToken=true
-    RefreshToken=true
-    AuthorizationCode=true
+UserClaims=true
+UserIdToken=true
+XACML_Request=true
+XACML_Response=true
+NTLM_Token=true
+SAML_Assertion=true
+SAML_Request=true
+AccessToken=true
+RefreshToken=true
+AuthorizationCode=true
```

Comment:
does the + icon show up as a bullet point? Just checking

Reply:
These aren't plus icons that I added. It's just github showing the latest change.